.github/workflows/ci.yml

```diff
@@ -52,10 +52,10 @@ jobs:
 
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
 
       - name: Setup node
-        uses: actions/setup-node@v3
+        uses: actions/setup-node@v4
         with:
           node-version: '18.18.0'
```

Comment:
Do you know if this override still needed?

Reply:
It is, they don't provide a default.
However, they support using an `lts` value for the `node-version`, so we could also go in that direction?

Reply:
`lts` lgtm for the purpose of this github action 👍

Reply:
Updated ✅